code/modules/integrated_electronics/subtypes/converters.dm

```diff
@@ -298,8 +298,8 @@
 	name = "string exploder"
 	desc = "This splits a single string into a list of strings."
 	extended_desc = "This circuit splits a given string into a list of strings based on the string and given delimiter. \
-	For example, 'eat this burger' will be converted to list('eat','this','burger'). Leave the delimiter null to get a list \
-	of every individual character."
+	For example, 'eat this burger' can be converted to list('eat','this','burger'). Leave the delimiter null to get a list \
+	of every individual character. Leave the delimiter to empty string to split on spaces."
```

Comment:
Not quite right, and I'm not sure how to phrase that properly. By default, the string is null. If one inputs only spaces, it will be sanitized to an empty string. Inputting an empty string will result in string being unchanged, so one has to "leave it to an empty string".